qiita_db/sql_connection.py

```diff
@@ -150,16 +151,33 @@ def _open_connection(self):
 
         try:
             self._connection = connect(**args)
-        except Exception:
-            # catch any exception and raise as runtime error
-            # attempt to point the user to their config file
-            error_text = ("Can't connect to database; user `%s` is not known. "
-                          "Your QIITA configuration file, located at `%s` must"
-                          " specify the correct user, via the `USER` parameter"
-                          " under the `[postgres]` header. Please refer to "
-                          "`INSTALL.md` in the QIITA base directory.")
-            error_text = error_text % (qiita_config.user, qiita_config.conf_fp)
-            raise RuntimeError(error_text)
+        except OperationalError as e:
+            # catch threee known common exceptions and raise runtime errors
+            try:
+                etype = e.message.split(':')[1].split()[0]
+            except IndexError:
+                # we recieved a really unanticipated error without a colon
+                etype = ''
+            if etype == 'database':
+                etext = ('This is likely because the database `%s` has not '
+                         'been created or has been dropped.' %
+                         qiita_config.database)
+            elif etype == 'role':
+                etext = ('This is likely because the user string `%s` '
+                         'supplied in your configuration file `%s` is '
+                         'incorrect or not an authorized postgres user.' %
+                         (qiita_config.user, qiita_config.conf_fp))
+            elif etype == 'Connection':
+                etext = ('This is likely because postgres isn\'t '
+                         'running. Check that postgres is correctly '
+                         'installed and is running.')
+            else:
+                # we recieved a really unanticipated error with a colon
+                etext = ''
+            ebase = ('An OperationalError with the following message occured'
+                     '\n\n\t%s\n%s For more information, review `INSTALL.md`'
```

Comment:
Minor, but could you add another `%s` between `e.message` and `etext`? I think having a blank line there will make the error message easier to read. This is not blocking if we want to get this PR merged now.